<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1351</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#Dialog{
background-color:white;
}</string>
  </property>
  <widget class="QLabel" name="user_png">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>user.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="employee_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 75 24pt &quot;League Spartan&quot;;</string>
   </property>
   <property name="text">
    <string>Abhinav Sen</string>
   </property>
  </widget>
  <widget class="QLabel" name="employeeIDHeading">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 75 16pt &quot;League Spartan&quot;;</string>
   </property>
   <property name="text">
    <string>Employee ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Emailaddress_heading">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 75 16pt &quot;League Spartan&quot;;</string>
   </property>
   <property name="text">
    <string>Email address:</string>
   </property>
  </widget>
  <widget class="QLabel" name="branchHeading">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 75 16pt &quot;League Spartan&quot;;</string>
   </property>
   <property name="text">
    <string>Branch: </string>
   </property>
  </widget>
  <widget class="QLabel" name="employeeIDLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 25 16pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>20CS10003</string>
   </property>
  </widget>
  <widget class="QLabel" name="emailIDLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 25 16pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>abhinavsen2002@gmail.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="branchLabel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>640</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
font: 25 16pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ujhv&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>160</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 24, 24);
font: 75 60pt &quot;League Spartan&quot;;
</string>
   </property>
   <property name="text">
    <string>TCCS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>290</y>
     <width>581</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(222, 24,24);
font: 75 28pt &quot;League Spartan&quot;;</string>
   </property>
   <property name="text">
    <string>Choose utility as required</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterConsignmentButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>500</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 75 20pt &quot;League Spartan&quot;;
color: white;
background-color: rgb(222, 24, 24);
border-radius: 15px
}
QPushButton:hover{
background-color: '#eeeeee';
color: rgb(222, 24, 24);
border: 2px;
border-color: rgb(222,24,24)
}</string>
   </property>
   <property name="text">
    <string>Enter Consigment</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>-270</x>
     <y>0</y>
     <width>811</width>
     <height>841</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>truck2.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>370</y>
     <width>671</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 15pt &quot;Calibri&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose Enter consigment to report a consigment order.&lt;br&gt; Choose the truck util to report consigment arrivals or manage truck data&lt;br&gt; in the branch&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="truckUtilButton">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>500</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 75 20pt &quot;League Spartan&quot;;
color: white;
background-color: rgb(222, 24, 24);
border-radius: 15px
}
QPushButton:hover{
background-color: '#eeeeee';
color: rgb(222, 24, 24);
border: 2px;
border-color: rgb(222,24,24)
}</string>
   </property>
   <property name="text">
    <string>Truck Utility</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
font: 75 20pt &quot;Calibri&quot;;
color: rgb(222, 24, 24);
background-color: white;
border-radius: 25px
}
QPushButton:hover{
background-color: &quot;#222222&quot;

}</string>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <zorder>label_10</zorder>
  <zorder>user_png</zorder>
  <zorder>employee_name</zorder>
  <zorder>employeeIDHeading</zorder>
  <zorder>Emailaddress_heading</zorder>
  <zorder>branchHeading</zorder>
  <zorder>employeeIDLabel</zorder>
  <zorder>emailIDLabel</zorder>
  <zorder>branchLabel</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>enterConsignmentButton</zorder>
  <zorder>label_3</zorder>
  <zorder>truckUtilButton</zorder>
  <zorder>backButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
